web/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/web/rest/dialog/DialogOppgaveController.kt

```diff
@@ -123,3 +156,19 @@ data class OpperettOppgaveRequest(
         val oppgaveTypeKode: String,
         val prioritetKode: String
 )
+
+data class OpperettSkjermetOppgaveDTO(
+        val fnr: String,
+        val valgtEnhetId: String,
+        val dagerFrist: Int,
```

Comment:
Denne snakket vi om å droppe fra frontenden og heller regne ut i backend? tanker om det?

Reply:
sidan frist er aktiv dato + prioritet, så er det mest praktisk å sette backend. Veileder velger prioritet og me setter aktiv dato til d.d og rekner ut fristen basert på prioriteten. Sidan fristen skal ta ein del hensyn til helg o.l, så er det mykje kode for lite når backenden har denne logikken allereie.

Reply:
ligger det kode i frontend som seier at HØY = 2?

Reply:
Nei, ligger som HOY, LAV, NORM

Reply:
`prioritetKode: "HOY_TEMA"`